src/components/IndvReview/IndvReview.jsx

```diff
@@ -33,11 +30,13 @@ export default function IndvReview() {
             <p className="indv-review-body">{review.review_body}</p>
             <p className="indv-review-author">
               <Link to={`/users/${review.owner}`}>By: {review.owner}</Link>
-            </p>
-                <p className="indv-review-votes">Votes: {review.votes}</p>
+                </p>
+                <PatchReviewVote review_id={review.review_id} votes={review.votes} />
+                {/* <p className="indv-review-votes">Votes: {review.votes}</p> */}
+
 
-                <h3 className="comment-title">Comments:</h3>
-            <Comments comment_id={review_id} />
+                {/* <h3 className="comment-title">Comments:</h3> */}
```

Comment:
Should not be any commented out code in your components

Reply:
This was commented out so I wasn't overstepping the bounds of what should have been done for this task, have uncommented it now and re-added comment functionality.